packages/calypso-e2e/src/lib/pages/media-page.ts

```diff
@@ -97,7 +99,11 @@ export class MediaPage {
 		await this.waitUntilLoaded();
 
 		await this.page.click( selectors.editButton );
-		await this.page.click( selectors.mediaModalEditButton );
+		await this.page.click(
+			`${ selectors.mediaModalActionBar( getTargetDeviceName() ) } ${
+				selectors.mediaModalEditButton
```

Comment:
I kept these two selectors separate (instead of lumping it into one dynamic selector) since:
1. this combination is used only once throughout the file, and;
2. one of the constituent selector is already a dynamic selector.